docs/editors/debug-adapter-protocol.md

```diff
@@ -57,6 +58,10 @@ Apart from using code lenses, users can start a debug session by executing the
 identically named classes in different modules. A uri will be returned that can
 be used by the DAP client.
 
+`envFile` is an optional parameter, which allows you to specify a path to a
+`.env` file with additional environment variables. It supports single line as
+well as multi-line quoted values (without value substitution).
```

Comment:
can we add how example file could look like to docs?
```
example.env:

# comment
key1=value1
key2='value2'    # comment too
key3="value3"
# key cannot contain dots or dashes
a.b.key4='value4'   # will be ignored
a-b-key5='value5'   # will be ignored
# multiline value
key6=' line1
       line2'
```

Just looked at tests and tried to extract essential things, if something else is nice, can be suprising might be worth adding here

Reply:
good idea, I added an example in 1ad5709